Sources/SwiftFrameCore/Config/DeviceData.swift

```diff
@@ -101,27 +98,28 @@ public struct DeviceData: Decodable, ConfigValidateable {
             throw NSError(description: "No screenshots were loaded, most likely caused by a faulty regular expression")
         }
 
-        try screenshotsGroupedByLocale.forEach { localeDict in
-            guard let first = localeDict.value.first?.value else {
-                return
-            }
-            try localeDict.value.forEach {
-                if let size = NSBitmapImageRep.ky_loadFromURL($0.value)?.ky_nativeSize, size != NSBitmapImageRep.ky_loadFromURL(first)?.ky_nativeSize {
-                    throw NSError(
-                        description: "Image file with mismatching resolution found in folder \"\(localeDict.key)\"",
-                        expectation: "All screenshots should have the same resolution",
-                        actualValue: "Screenshot with dimensions \(size)")
-                }
-            }
+        guard numberOfSlices > 0 else {
+            throw NSError(
+                description: "Invalid numberOfSlices value",
+                expectation: "numberOfSlices value should be >= 1",
```

Comment:
I know this means the same as the actual condition, but to me it would make sense to have the condition and the expectation's textual description be the exact same

Reply:
I think this follows the pattern of `NSError`, it has `localizedFailureReason`, `localizedDescription` and `localizedRecoverySuggestion` which this kind of mimics a bit 🤔

Reply:
What I mean is that I find it odd that the condition is `numberOfSlices > 0` but then the textual description is `numberOfSlices value should be >= 1` instead of it being `numberOfSlices value should be > 0` :D